README.md

```diff
@@ -160,6 +162,14 @@ Results for applying pruning to reduce the parameters of the following models by
 | DenseNet121 [3] | 74.44                | 71.71                 |
 
 
+## Quantization Trouble Shooting
+
+If the accuracy of the quantized model is too large for your application, check out the [Quantization Troubleshooting](https://github.com/sony/model_optimization/tree/main/quantization_troubleshooting.md)
+for common pitfalls and some tools to improve quantization accuracy.
+
+## FAQ
+
+Check out the [FAQ](https://github.com/sony/model_optimization/tree/main/FAQ.md) for common issues.
```

Comment:
combine to one section: ## Troubleshooting
Note the the typo: If the accuracy of the quantized model is **too large**